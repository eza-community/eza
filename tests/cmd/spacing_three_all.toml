bin.name = "eza"
args = "tests/itest --grid --spacing 3"
