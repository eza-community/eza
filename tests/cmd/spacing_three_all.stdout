a
b
c
d
dir-symlink -> vagrant/debug
e
exa
f
g
h
i
image.jpg.img.c.rs.log.png
index.svg
j
k
l
m
n
o
p
q
vagrant
